I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -1596,7 +1609,7 @@ Line |     .  +-----+ | |  *  *  | | +-----+  .     | Line
   
   <t>
    In case of individual OTSi protection, there are two network media
-   channel paths associated with the OTSi signal. In the YANG model,
+   channel (NMC) paths associated with the OTSi signal. In the YANG model,
```

Comment:
This is not consistent with the NMC definition in ITU-T Rec. G.807 Amendment 1! See also #122

Reply:
See #125 - draft text updated:
- "network media channel (NMC)" replaced with "end-to-end MC"
- nmc-path-id replaced with e2e-mc-path-id